Bixal/move.mil
# Feature/156844285/parse 400 ng file

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Address pivotal ticket #156844285
- Writes zip3.json, zip5.json, entitlements.json, linehauls.json, shorhauls.json, packunpack.json, and service_areas.json for the PPM tool.

## Testing

To verify the changes proposed in this pull request…

1. Pull branch changes, do a `composer install`, `make cim`.
1. Since there was a Drupal core update, execute `docker-compose run php drupal updb` to fix the external links table (it'll be broken otherwise).
1. Execute `docker-compose run php drupal parser` choose zip3.
1. Check that the json file is generated at `web/sites/default/files/tools/data`.

See https://github.com/Bixal/move.mil/pull/107 for reference
